src/app/visual/sectioned-form/sectioned-form-visual.component.ts

```diff
@@ -1,10 +1,23 @@
 import { ChangeDetectionStrategy, Component} from '@angular/core';
 
+import {
+  SkyThemeService,
+  SkyThemeSettings
+} from '@skyux/theme';
+
 @Component({
   selector: 'sectioned-form-visual',
   templateUrl: './sectioned-form-visual.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush
 })
 export class SectionedFormVisualComponent {
   public maintainSectionContent: boolean = false;
+
+  constructor(
```

Comment:
There are some odd spacing things going on in this block. Can we bump the closing constructor parenthesis onto a new line and unindent the `themeSettingsChange` declaration?

Reply:
Addressed in recently commit. https://github.com/blackbaud/skyux-tabs/pull/178/commits/f23bd3b82e0f0bcaa9face16e77ee5d8720a483d